### PR TITLE
Mention @Security() annotation instead of @Secure()

### PR DESCRIPTION
**Pull request by javiereguiluz**
| Q | A |
| --- | --- |
| Doc fix? | no |
| New docs? | yes |
| Applies to | 2.4+ |
| Fixed tickets | - |

Beware that (if accepted) this PR must be merged in 2.4+ because `@Security` isn't available in Symfony 2.3.

### CODE REVIEW

**Comment by xabbuh** [comment]
I think we should simply merge this into the `2.5` branch.

By the way, you would have to revert the `2.6` specific changes.

File: cookbook/security/remember_me.rst [hunk 1]
```diff
@@ -171,21 +171,25 @@ In the following example, the action is only allowed if the user has the
...
+     * @Security("has_role('IS_AUTHENTICATED_FULLY')")
      */
     public function editAction($name)
     {
         // ...
     }
 
+.. versionadded:: 2.4
+    The ``@Security`` annotation was introduced in SensioFrameworkExtraBundle 3.0,
+    which can only be used with Symfony 2.4 or later.
```
Comment (stof) [inline cookbook/security/remember_me.rst]: this is not true technically. Other features of SensioFrameworkExtraBundle 3.0 can be used in Symfony 2.3

Comment (javiereguiluz) [inline cookbook/security/remember_me.rst]: I copied this note from this other note:

```
The @Security annotation was introduced in SensioFrameworkExtraBundle 3.0. This bundle 
version can only be used with Symfony 2.4 or later (see the SensioFrameworkExtraBundle 
release cycle).
```

Source: http://symfony.com/doc/current/bundles/SensioFrameworkExtraBundle/annotations/security.html

Should we also fix the original note?

Comment (xabbuh) [inline cookbook/security/remember_me.rst]: As I said [here](https://github.com/symfony/symfony-docs/pull/4735#issuecomment-68540224), we shouldn't over-complicate things here. Even the bundle's readme just states "The minimum version of Symfony2 for this workflow is 2.4.0.".

**Comment by javiereguiluz** [comment]
@xabbuh is there anything else left to be done in this PR before consider it *finished*? Thanks.

**Comment by xabbuh** [comment]
@javiereguiluz Given that 2.5 has reached the end of maintenance in the meantime, I think you can put back the 2.6 related changes. So this can then be merged into the `2.6` branch directly.

**Comment by javiereguiluz** [comment]
@xabbuh I've done what you said, so this PR is ready for the final review. Thanks.

**Comment by xabbuh** [comment]
:+1:

**Comment by WouterJ** [comment]
@javiereguiluz could you please rebase this PR against 2.6? GH has lots of trouble with our submodules when switching form an unmaintained branch (2.4) to a maintained one.

**Comment by javiereguiluz** [comment]
Closed this pull request in favor of #4957

**Pull request closed without merging**